### ISSUE TITLE
Modification: Project page, cards, filters, and modal cleanups and changes - figma

### ISSUE THREAD

**Issue by Ifycode** [code-collabo/collabo-web-app#26]
#### Ticket description
A lot of updates have been made to the code collabo community as a whole, its projects and documentation. This updates/changes will therefor have to be reflected in the collabo web app.

#### Changes to make to web app design based on community updates
- We no longer use "standalone", "parent or child projects". All we have/use now is the project name, and the number of projects under it: 1 projects, 2 projects, 3 projects etc. Should this info bring up a modal as before? We will discuss this.
- Issue buttons on project cards and modals: We now have a centralized way of accessing issues. So having issue button on each project card will no longer be necessary. How can we send contributors to our project boards once on the page?
- Repo readme button on project cards and modals: Do we still need a modal? Maybe all we need is a "Contribute to project" button on the cards that will take contributor to the contributing doc page for that project.

#### In summary
- [x] Project cards will only have the following:
    - The image
    - Project name
    - Number of projects (this will no longer be links)
    - "Contribute to project" button
- [x] Modals: We may no longer use the modal for showing details, but we may use the concept when its time for working on the edit aspect of the projects.
